Build/upload.ps1

```diff
@@ -1,13 +1,13 @@
 param([string]$apiKey)
 
-Invoke-WebRequest -Uri "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe" -OutFile "nuget.exe"
+#Invoke-WebRequest -Uri "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe" -OutFile "nuget.exe"
 
-$files = Get-ChildItem -Path ".\NuGet" -Filter "*.nupkg"
-foreach ($file in $files)
-{
-	Write-Host "Uploading: " $file
+#$files = Get-ChildItem -Path ".\NuGet" -Filter "*.nupkg"
```

Comment:
Why is this commented out?

Reply:
Life sucks :-) I just forgot to remove the comment, I did not want to upload anything during my local development ... sorry for that

Reply:
OK then please remove the changes.